projects/vdk-plugins/vdk-meta-jobs/.plugin-ci.yml

```diff
@@ -3,20 +3,20 @@
 
 image: "python:3.7"
 
-.build-vdk-meta-jobs:
+.build-vdk-dags:
```

Comment:
may be vdk-dag

Reply:
vdk-dags is an obscure form of multiple DAGs..someone may wonder if DAGs or DAGS (with another meaning).
I think vdk-dag is more simple, and self-explanatory.